protoc-gen-grpc-gateway/descriptor/types.go

```diff
@@ -451,7 +451,16 @@ var (
 	}
 
 	wellKnownTypeConv = map[string]string{
-		".google.protobuf.Timestamp": "runtime.Timestamp",
-		".google.protobuf.Duration":  "runtime.Duration",
+		".google.protobuf.Timestamp":   "runtime.Timestamp",
```

Comment:
@achew22 @johanbrandhorst These values don't really exist in [runtime](https://github.com/grpc-ecosystem/grpc-gateway/tree/master/runtime) package. 

So this change only helps in generating reverse proxy (gw.go code) files. It's not sufficient to support "adding protobuf wrappers in url template option" (https://github.com/grpc-ecosystem/grpc-gateway/issues/808).

Reply:
I will revert this MR until we have a proper fix in place then.

Reply:
Wait, what do you mean? These functions do exist in the runtime:

https://github.com/grpc-ecosystem/grpc-gateway/blob/master/runtime/convert.go#L16

Reply:
StringValue, BytesValue, Int32Value... etc don't.

Reply:
It'll work.But I'm afraid it'll defeat the purpose of using `StringValue` in the first place. It'll just look like we're providing support for `StringValue` but internally we'll just be giving them `String`.
Correct me if I'm wrong.

Reply:
We intentionally treat `google.protobuf.StringValue` as a string. That's a feature!

Reply:
Oh! If that's the case I'll check if I can make it work.

Reply:
I'm going to revert this since it's broken. Please submit a separate PR.

Reply:
Yes makes sense.